server/server.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
```

Comment:
why use `require` instead of `import`?

Reply:
Express is run through Node.js, that doesn't support `import` syntax by default, the only way to change this is by ejecting the Webpack config, which, if I recall well, you wouldn't  really want to, haha!